src/components/Modal/Modal.vue

```diff
@@ -750,6 +758,17 @@ $header-size: 50px;
 			height: 35vw;
 		}
 	}
+
+	// Make modal always full screen on mobile independent from modal size
+	@media only screen and (max-width: $breakpoint-mobile/2) {
```

Comment:
Why breakpoint-mobile/2 ?
We never use that anywhere else on Nextcloud

Reply:
Thats incorrect because there is actually already one place, in the same file: https://github.com/nextcloud/nextcloud-vue/blob/69ca1d83ef367ef985d039e3d73843c165bc132a/src/components/Modal/Modal.vue#L578

The problem is that breakpoint-mobile=1024 which is far too much but around 500px is the perfect breakpoint for making all modals full screen -> hence i've done it this way.

Reply:
Should we just reduce the breakpoint then?

Reply:
Honestly fine by me but would be another breaking change ;)
@skjnldsv should decide if it is wort the trouble!
(Would also need to be changed in the ismobile mixin and other places where breakpoint-mobile is used)

Reply:
let's leave like that, I should have commented my code, sorry for that :)
Nice catch @szaimen !